scripts/check_frontend_coverage.py

```diff
@@ -246,7 +246,7 @@
     'profile-link-text.component.ts',
     'progress-nav.directive.ts',
     'promo-bar.directive.ts',
-    'promo-bar.service.ts',
+    'promo-bar-backend-api.service.ts',
```

Comment:
This line (`promo-bar-backend-api.service.ts`) will now come before `promo-bar.directive.ts` as "-" has an ASCII value if 45 and "." has an ASCII value of 46. This is the reason why the frontend tests are failing for this PR as this list has to be sorted lexicographically.

Reply:
Done!